Src/EmbeddedScripts.CSharp.MonoEvaluator/EvaluatorExtension.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Reflection;
+using Mono.CSharp;
+
+namespace EmbeddedScripts.CSharp.MonoEvaluator
+{
+    internal static class EvaluatorExtension
+    {
+        public static ReflectionImporter GetReflectionImporter(this Evaluator evaluator) =>
```

Comment:
nit: as it's used in a 1 single place, it can be an EvaluatorExtension instead, then you don't see internal type ReflectionImporter in the MonoCSharpRunner code